core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -799,7 +812,8 @@ public void deleteInode(LockedInodePath inodePath, long opTimeMs)
    * @throws FileDoesNotExistException if the Inode cannot be retrieved
    */
   public void deleteInode(LockedInodePath inodePath) throws FileDoesNotExistException {
```

Comment:
the only caller of this method is from tests, I suggest we inline it there and remove it here to avoid production code accidentally using this method not realizing it does not journal